host/xtest/pkcs11_1000.c

```diff
@@ -2565,3 +2565,237 @@ static void xtest_pkcs11_test_1011(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1011, xtest_pkcs11_test_1011,
 		 "PKCS11: Test Find Objects");
+
+
+static void xtest_pkcs11_test_1012(ADBG_Case_t *c)
+{
+	CK_RV rv = CKR_GENERAL_ERROR;
+	CK_SLOT_ID slot = 0;
+	CK_SESSION_HANDLE session = CK_INVALID_HANDLE;
+	CK_SESSION_HANDLE ro_session = CK_INVALID_HANDLE;
+	CK_FLAGS session_flags = CKF_SERIAL_SESSION | CKF_RW_SESSION;
+	CK_FLAGS ro_session_flags = CKF_SERIAL_SESSION;
+	CK_OBJECT_HANDLE obj_hdl = CK_INVALID_HANDLE;
+	const char *label = "Dummy Objects";
+	bool logged_in = false;
```

Comment:
maybe rename `logged_in` to `ro_session_logged_in` ?

Reply:
Changed